README.md

```diff
@@ -58,12 +58,12 @@ or it can be integrated with Docker:
 # init-based
 sudo service docker stop
 # systemd-based
-sudo systemctl stop docker
+sudo systemctl stop docker.sock
```

Comment:
I think that the service name is called `docker` so you can use `sudo systemctl stop docker` or `sudo systemctl stop docker.service`. I've tested it on Ubuntu 20, should be okay with `docker`

Reply:
The latest version of docker comes with that kind of specification, you have used this one?

Reply:
Actually didn't test with the latest version, I'll give it a try. Seems straight forward what to do here 😄